spec/CookiePluginSpec.php

```diff
@@ -146,7 +146,12 @@ function it_saves_cookie(RequestInterface $request, ResponseInterface $response,
 
         $response->hasHeader('Set-Cookie')->willReturn(true);
         $response->getHeader('Set-Cookie')->willReturn([
-            'cookie=value',
+            sprintf(
+                'cookie=value; expires=%s; Max-Age=60; path=/; domain=test.com; secure; HttpOnly',
+                (new \DateTime())->add(
+                    new \DateInterval('PT60S')
```

Comment:
is this in seconds? will that mean the test fails if between this and the execution, the internal clock moves to the next second? that would be a flaky test. maybe we should regex match to allow the next second as well (i guess its reasonable to assume that the test will complete within 1 second at max) something like (%s|%s) with  \DateInterval('PT60S') and  \DateInterval('PT61S')